bin/improver-nbhood

```diff
@@ -41,18 +41,33 @@ from improver.nbhood import BasicNeighbourhoodProcessing
 def main():
     """Load in arguments and get going."""
     parser = argparse.ArgumentParser(
-        description='Apply basic weighted circle smoothing via ' +
-                    'the BasicNeighbourhoodProcessing plugin ' +
-                    'to a file with one cube.')
-    parser.add_argument('--radius-in-km', metavar='RADIUS', type=float,
-                        help='The kernel radius for neighbourhood processing')
+    description='Apply basic weighted circle smoothing via ' +
+                'the BasicNeighbourhoodProcessing plugin ' +
+                'to a file with one cube.')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--radius-in-km', metavar='RADIUS', type=float,
+                       help='The kernel radius for neighbourhood processing')
+    group.add_argument('--radii-in-km-by-lead-time',
+                       metavar='RADIUS_BY_LEAD_TIME', nargs=2,
+                       help='The kernel radii for neighbourhood processing '
+                       'and the associated lead times at which the radii are '
+                       'valid.')
```

Comment:
At present when called from the command line this can only accept a single pair of time varying values (one radius, one lead time).